app/models/ext_management_system.rb

```diff
@@ -603,6 +603,10 @@ def tenant_identity
     User.super_admin.tap { |u| u.current_group = tenant.default_miq_group }
   end
 
+  def top_level_manager
```

Comment:
Why are managers nested?  Wouldn't you just query the `Provider` for its other managers?